twilight-lavalink/src/node.rs

```diff
@@ -539,6 +552,71 @@ impl Connection {
         Ok(())
     }
 
+    fn get_outgoing_endpoint_based_on_event(
+        &mut self,
+        outgoing: &OutgoingEvent,
+    ) -> Result<(Method, hyper::Uri), NodeError> {
+        let address = self.config.address;
+        tracing::debug!("forwarding event to {address}: {outgoing:?}");
```

Comment:
Please keep capitalization at log start consistent

Reply:
they are all lower case is that not what you mean?